profile/README.md

```diff
@@ -0,0 +1,11 @@
+[![Accelerate your robotics development](https://user-images.githubusercontent.com/14011012/195918769-5aaeedf3-5de2-48fb-951e-7399f2b9e190.png)](https://foxglove.dev)
+
+[<img align="right" width="400" src="https://github.com/foxglove/studio/raw/main/resources/screenshot.png">](https://github.com/foxglove/studio)
+
+Foxglove was founded to improve the development experience for robotics teams by bringing high quality tools to the robotics industry. Our mission is to reduce the time to market for our users, and accelerate the robotics revolution.
```

Comment:
"high-quality"

We use "robotics" 3 times in 2 sentences - could pare down.

Suggestion: "Foxglove provides high-quality tools that streamline the robotics development experience. We hope to accelerate the robotics revolution by reducing the time to market for our users."

Reply:
I took this content directly from https://foxglove.dev/about.

Reply:
Oof got it - will revisit that page.